Doc/whatsnew/3.12.rst

```diff
@@ -850,7 +850,7 @@ New Features
   The :const:`Py_TPFLAGS_HAVE_VECTORCALL` flag is now removed from a class
   when the class's :py:meth:`~object.__call__` method is reassigned.
   This makes vectorcall safe to use with mutable types (i.e. heap types
-  without the :const:`immutable <Py_TPFLAGS_IMMUTABLETYPE>` flag).
+  without the immutable flag, :const:`Py_TPFLAGS_IMMUTABLETYPE`).
```

Comment:
Likewise, this _should_ be documented as a `c` constant, not a Python constant...but not worth making that here, I guess. However, does need a punctuation fix (no comma here):


```suggestion
  without the immutable flag :const:`Py_TPFLAGS_IMMUTABLETYPE`).
```

Reply:
Ah, yes, of course, I must have misread—serves me right for reviewing this at nearly 5 am, heh. Thanks for the catch!

(@merwok BTW, I just emailed the trademarks committee about an unrelated matter—just wanted to make sure it didn't go to spam or anything).

Reply:
I don’t know why you’re tagging me about trademarks!

Reply:
You want `@malemburg` for trademarks, not `@merwok`.

Reply:
Oops, somehow got confused, sorry—my bad!